README.md

```diff
@@ -14,9 +14,16 @@ Vorta is a backup client for macOS and Linux desktops. It integrates the mighty
 
 
 ## Installation
-Vorta should work on all platforms that support Qt and Borg. This includes macOS, Ubuntu, Debian, Fedora, Arch Linux and many others. Windows is currently not supported by Borg, but this may change in the future.
+Vorta should work on macOS and all Linux distributions that [support Flatpak](https://flatpak.org/setup/). Windows is currently not supported by Borg, but this may change in the future.
```

Comment:
Flatpak is not available on all Linux distros out of the box. On Ubuntu I had to install some extra KDE Flatpaks first. And Flatpak itself of course.

The lightest and most universal install option is still Pip. So "should work on [...] all Linux distributions that support Flatpak" is misleading, because it's not strictly required at all.

There is also no reason to split the install instructions up like this. They already have their own file.

Reply:
I am quite surprised by your response. So let me bring up some counter arguments.

> Flatpak is not available on all Linux distros out of the box. 

So is pip, but for flatpak we provide clear installation instructions compared to pip where we leave the user alone to find out they have to execute e.g. sudo apt install python3-pip beforehand.

`On Ubuntu I had to install some extra KDE Flatpaks first`

Yes, but that should happen automatically.

`The lightest and most universal install option is still Pip.`

Lightest yes, most universal? Also true, but is that so important when [this](https://flatpak.org/setup/) includes pretty much every commonly used Linux distribution?

> So "should work on [...] all Linux distributions that support Flatpak" is misleading, because it's not strictly required at all.

I agree with that one, haven't thought of that.

> 
There is also no reason to split the install instructions up like this. They already have their own file.


Yeah... But I would still argue that we should recommend certain kind of installation methods. One way or the other. I just want to note hear that after two years of using python I still had to search the internet just to install Vorta.
On Ubuntu you have to:

- find out how to install pip of python 3
- use pip3
- live with a version that doesn't update itself and has no icon

On Fedora:

- you have pip but get a mysterious error message when you try to install
- okay the internet says I have to install python3-devel
- now people might get an error again
- they have to think about using pip3 not pip and don't use sudo, but use the --user flag instead

And just think how many pip-related issues we had by now.

Reply:
I now pushed an alternative to this pull request,

Reply:
I like the new version, but let's still keep it open for a while and see if all makes sense.

One reason why I didn't want to emphasize Flatpak too much is because it's a new feature. If some people find it, great. If there are any problems and too many find it, bad. Better keep it low for a while and emphasize it more over time.

I'm aware that Flatpak solves many common install issues (possibly the new #312?) and that it can reach a wider audience of non-tech people than `pip`. It could become our main distribution channel for Linux. We just need to be aware of the complexity involved and there will be improvements over time. E.g. I didn't get menu icons in after installing it on XFCE.

Once we trust it sufficiently, we can emphasize it more on the Github frontpage. Flathub will also help to discover new users to find Vorta. Especially the large group that doesn't install stuff from Github directly. (I actually made a landing page + static docs for Vorta some months ago, but never finished it. Mostly because I wasn't sure if we have the ease-of-use level required for a less technical audience.)

Reply:
> One reason why I didn't want to emphasize Flatpak too much is because it's a new feature. If some people find it, great. If there are any problems and too many find it, bad. Better keep it low for a while and emphasize it more over time.

That is a very good point. I agree, let's keep it low for a while.

> E.g. I didn't get menu icons in after installing it on XFCE.

Is it the same problem as #277? Because that one was not flatpak-specific.